var/spack/repos/builtin/packages/librsb/package.py

```diff
@@ -11,9 +11,10 @@ class Librsb(AutotoolsPackage):
     library for the Recursive Sparse Blocks format"""
 
     homepage = "http://librsb.sourceforge.net/"
-    url      = "http://download.sourceforge.net/librsb/librsb-1.2.0.9.tar.gz"
+    url      = "http://download.sourceforge.net/librsb/librsb-1.2.0.10.tar.gz"
```

Comment:
Generally it's OK not to tweak the URL when providing a new version, since spack just parses the URL as a "template" rather than using it directly.